tests/phpunit/MultiInstanceFileTestCase.php

```diff
@@ -0,0 +1,688 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of vfsStream.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace bovigo\vfs\tests;
+
+use bovigo\callmap\NewInstance;
+use bovigo\vfs\content\StringBasedFileContent;
+use bovigo\vfs\MultiInstanceFile;
+use bovigo\vfs\vfsStreamFile;
+use bovigo\vfs\vfsStreamWrapper;
+use PHPUnit\Framework\TestCase;
+use const SEEK_CUR;
+use const SEEK_END;
+use const SEEK_SET;
+use function bovigo\assert\assertThat;
+use function bovigo\assert\predicate\equals;
+use function bovigo\assert\predicate\isSameAs;
+use function bovigo\callmap\verify;
+use function rand;
+use function strlen;
+use function uniqid;
+
+/**
+ * Test for bovigo\vfs\MultiInstanceFile.
+ */
+class MultiInstanceFileTestCase extends TestCase
```

Comment:
Since `MultiInstanceFile` is a proxy to `vfsStreamFile` could we reuse the test case?

Reply:
Like, make this extend vfsStreamFileTestCase? That would likely break. `MultiInstanceFile` isn't a complete proxy of `vfsStreamFile`. It only proxies the methods being called inside `vfsStreamWrapper` because that's the only place this is (should be?) used.